apps/agent/buidler.config.js

```diff
@@ -0,0 +1,27 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin("@nomiclabs/buidler-truffle5")
+usePlugin('buidler-gas-reporter')
+usePlugin('solidity-coverage')
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
```

Comment:
I'm wondering if it'd be worth adding some default networks like rinkeby and mainnet? I've still not quite figured out how to do the account management bit with buidler.

Reply:
Good point, I'll include a few network configurations.